Jim-Hodapp-Coaching/refactor-platform-fe
# Use the free Material-UI Devias.io template as a baseline frontend

## Description
This PR introduces the free Material-UI template Devias.io, customizes the login screen text, and removes other remnants of tailwindcss.


#### GitHub Issue: Closes #5 

### Changes
* Adds the free Material-UI template Devias.io
* Customizes the text on the login screen
* Finishes removing the last tailwindcss remnants

### Screenshots / Videos Showing UI Changes (if applicable)

<img width="1676" alt="Screenshot 2023-11-18 at 15 28 20" src="https://github.com/Jim-Hodapp-Coaching/refactor-platform-fe/assets/3219120/89a2c785-5e62-4a82-933a-02587adee176">


### Testing Strategy
1. Make sure you're using node@21 or above (take a look at using [Node Version Manager](https://github.com/nvm-sh/nvm), nvm to manage your local Node.js versions)
2. `npm run dev`
3. Browse to http://localhost:3000


### Concerns
None
